weed/s3api/s3api_xsd_generated_helper.go

```diff
@@ -0,0 +1,10 @@
+package s3api
+
+type Grantee struct {
```

Comment:
The `xsdgen -o s3api_xsd_generated.go -pkg s3api AmazonS3.xsd` will generate an empty Grantee struct.

Need to adjust the readme steps to remove the generated struct. (I do not remember how it worked before)